lib/_tls_wrap.js

```diff
@@ -858,7 +858,8 @@ exports.connect = function(/* [port, host], options, cb */) {
 
   var hostname = options.servername ||
                  options.host ||
-                 options.socket && options.socket._host,
+                 options.socket && options.socket._host ||
```

Comment:
Generally we prefer shorter I think, but no harm in adding a few clarity parens, I'd say.